python/integrationtests-basic

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+import unittest
+
+class SimulationTestCase(unittest.TestCase):
+    def setUp(self):
+        import signal
+
+        self.expected = self.expected_session_time()
+        self.timeout = int(self.expected * 1.5 + 1.5) + 1
+
+        signal.alarm(self.timeout)
+
+        from dronin import telemetry
+
+        if self.should_wipe_first():
+            import os
+
+            try:
+                os.remove('build/unittest.flash')
+            except FileNotFoundError:
+                pass
+
+        args = [ "-c", "./build/flightd/flightd -! -S telemetry:stdio -c build/unittest.flash" ]
+        t_stream = telemetry.get_telemetry_by_args(service_in_iter=False,
+                arguments = args)
+        t_stream.start_thread()
+        t_stream.wait_connection()
+
+        self.stream_iter = iter(t_stream)
+        self.t_stream = t_stream
+
+    def tearDown(self):
+        self.t_stream._close()
+
+        import signal
+
+        remaining = signal.alarm(0)
+        #print("Elapsed %d vs expected %f\n"%(self.timeout - remaining, self.expected))
+
+    def wait_for_tick(self):
+        for o in self.stream_iter:
+            if o.name=='UAVO_HwSimulation':
+                if o.FakeTickBlocked != 0:
+                    return True
+
+        # This is bad.  Got to end of stream, no tick.
+        return False
+
+    def expected_session_time(self):
+        return 9.0
+
+    def should_wipe_first(self):
+        return False
+
+class AAConfTests(SimulationTestCase):
+    def should_wipe_First(self):
+        return True
+
+    def testDoConfig(self):
```

Comment:
test_config ? The python `unittest` docs use lowercase with underscores as sep

Reply:
Can do so.  I'm conflicted on what to do with this stuff, because I'm subclassing from something that usesThisCase. :/

Reply:
Only because the class predates PEP8. All the test cases in docs are PEP8 compliant.